README.md

```diff
@@ -45,6 +45,18 @@ gatsby develop
 
 The site is now running at `http://localhost:8000`!
 
+You can also use Docker compose: 
+
+    $ docker compose up
```

Comment:
With my installation, I had to use `sudo docker-compose up` (thanks for the pointer @dr0i).

Should we document different usages here or require some specific minimum docker version?

Reply:
with the mentioned links in the README you would be led to installation of `docker compose`. 

I think `docker-compose` will sooner or later be removed in favor of `docker compose`. This comment notes it as deprecated: https://stackoverflow.com/a/66516826/8896235
This article seems to confirm that: https://www.docker.com/blog/new-docker-compose-v2-and-v1-deprecation/

All the links on the `docker-compose` (v1) PyPi package are also already leading to the new `docker compose` repo (v2).

Reply:
Thanks, @sroertgen 

Regarding the use of sudo or not: if you did the [post-installation steps](https://docs.docker.com/engine/install/linux-postinstall/) and add the current user to the docker group you won't have to sudo, i think.